### PR TITLE
Use PluginShare to share the keepalive state

### PR DESCRIPTION
**Pull request by Hzj-jie**
Currently the "checked" mark does not work.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7416)

### CODE REVIEW

**Comment by Hzj-jie** [comment]
@Frenzie wanna take a quick look? Thank you.

**Comment by Hzj-jie** [comment]
By the way, maybe we can change the name of this plugin into "demo mode".

File: plugins/keepalive.koplugin/main.lua [hunk 1]
```diff
@@ -18,17 +19,16 @@ local function showConfirmBox()
         cancel_text = _("Close"),
         cancel_callback = function()
             disable()
-            menuItem.checked =false
+            PluginShare.keepalive =false
```
Comment (Frenzie) [inline plugins/keepalive.koplugin/main.lua:22]: nit

**Review by Frenzie — APPROVED** [review]
I think that should be fine.

File: plugins/keepalive.koplugin/main.lua (hunk 1 above)
Comment (Hzj-jie) [reply plugins/keepalive.koplugin/main.lua]: oops

**Pull request merged**